Utils/TimeConstants.swift

```diff
@@ -79,6 +79,43 @@ extension NSDate {
 
         return String(format: NSLocalizedString("just now", comment: "Relative time for a tab that was visited within the last few moments."))
     }
+
+    public func toRelativeTimeConciseString() -> String {
```

Comment:
This seems almost exactly the same as the method above. Is there any way we can combine them? Maybe adding a concise flag we pass in to rejig the string we get back?